nvi

```diff
@@ -1,12 +1,21 @@
 #!/bin/bash
 
+set -o errexit
+set -o nounset
+set -o pipefail
+
 NVI_VERSION=1.0.0
 B=$(tput bold) # Bold
 N=$(tput sgr0) # Normal
 U=$(tput smul) # Underline
 
-function print_usage() {
-  cat << EOF
+print_usage() {
+  less \
+    --raw-control-chars \
```

Comment:
Okay, I'm just going to not change this at all. I was going to replace it with `read -r d ''` but `-d ''` doesn't work with `set -o errexit`. You can work around that, but then I discovered `read` only reads one byte at a time. That makes sense but there are over 3000 here. It ends up being as slow as forking.